VM/rel2vm_params.py

```diff
@@ -611,10 +612,11 @@ def optimise_vm_params(
         fault_depth = srf_meta["hdepth"]
 
     rjb = 0
+    # TODO: Make this conditional useful. Currently always True
```

Comment:
not quite. It's a different rrup. 

It's the distance that is returned from the pgv threshold - for that calculation it assumes you have a surface point source. 

It looks like otherwise, rjb = 0 which seems like a bug. I think we need a good flowchart made for this logic - I think it was originally scoped to have been revisited in the refactor.

Reply:
Yeah, good catch, I realised it was a different rrup when I got home.

Definitely agree on this needing a flowchart/actual refactor